lib/orbf/rules_engine/builders/activity_formula_variables_builder.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
-
+require_relative "./substitution_builder"
```

Comment:
I would keep this in lib/orbf/rules_engine.rb (it's the only place where we do this)